pyment.conf

```diff
@@ -0,0 +1,6 @@
+first_line = false
+quotes = '''
```

Comment:
```suggestion
quotes = """
```

Reply:
For `ruff`.

Reply:
this is part of the pyment config. I'll fix

Reply:
oh nvm